src/version.ts

```diff
@@ -1,22 +1,20 @@
 import * as child_process from 'child_process';
 
-let version = "Unknown";
+let version = process.env.npm_package_version ? "v" + process.env.npm_package_version : "Unknown";
```

Comment:
This looks fine, though I'm not sure it'll work for the Docker container.

It currently runs: `NODE_ENV=production exec node build/app/index.js`

Because it's bypassing npm, it won't have a version variable.

Reply:
🤦🏻‍♂️ You're right, I don't know why I thought node would also take that environment variable. I have to find a different way

Reply:
if it's the Docker image putting the package.json at an inconvenient place, I'd just move it to the correct spot :p

Reply:
"npm run start:app" unfortunately would also fail because package.json is at the wrong location since it tries to look in build/package.json. It is not only a Docker container issue. I think moving the package.json file isn't the best way to do 🤔

Reply:
See the root comment below